Textile/Classes/Services/RequestsHandler.h

```diff
@@ -6,13 +6,14 @@
 //
 
 #import <Foundation/Foundation.h>
-#import <Mobile/Mobile.h>
+#import "TextileApi.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface RequestsHandler : NSObject <CoreCafeOutboxHandler>
 
-@property (nonatomic, strong) MobileMobile *node;
+- (instancetype)initWithTextile:(Textile *)textile;
+- (void)start;
```

Comment:
We need to be able to init the `RequestsHandler` early because it required to create the node. But we don't want to create it's `NSURLSession` until after the node is created so that the node will exist to handle any `NSURLSession` update events from the OS. I do this by moving the session creation code into `start` and calling it using that `dispatch_async` above in `TextileApi.m`.